core/src/main/java/io/questdb/cairo/mv/MatViewRefreshJob.java

```diff
@@ -254,7 +254,7 @@ private boolean insertAsSelect(
                                 .put("' not found in view select query");
                     }
 
-                    try (RecordCursor cursor = factory.getCursor(mvRefreshExecutionContext)) {
+                    try (RecordCursor cursor = factory.getCursor(refreshExecutionContext)) {
```

Comment:
method can do with restructure, there is a warning below that `copier.copy` might produce an NPE - lets not ignore this

Reply:
Added null-check assertions in d914617fa373d60e9a989fa9ddbc4a9839b2ea9a